app/src/main/res/values/strings.xml

```diff
@@ -214,7 +214,7 @@
   <string name="patientsummary_toolbar_title">%1$s, %2$s, %3$s</string>
   <string name="patientsummary_bangladesh_national_id">ID</string>
 
-  <string name="patientsummary_contact_doctor" translatable="false">Tele-consult</string>
+  <string name="patientsummary_teleconsult">Tele-consult</string>
```

Comment:
Since we are now adding the `patientsummary_teleconsult` string in the first commit, we can combine the last 3 commits into one single commit called `Remove unused resources`.